dex/src/main/java/io/crate/data/join/HashInnerJoinBatchIterator.java

```diff
@@ -70,7 +70,9 @@
 
     private static int DEFAULT_BUFFER_SIZE = 10_000;
     private final Predicate<C> joinCondition;
-    private final Multimap<Integer, Object[]> buffer;
+    private final com.carrotsearch.hppc.IntObjectHashMap<Object[]> buffer;
+    private final com.carrotsearch.hppc.IntObjectHashMap<List<Object[]>> bufferWithHashCollisions;
```

Comment:
Why is it necessary to specify the class fully qualified?

Reply:
not necessary, should be imported.